docs/articles/licensing/publishing-binaries.rst

```diff
@@ -0,0 +1,30 @@
+Publishing binary images
+========================== 
+
+We release binary images of our products that are licensed under the 
+[https://www.gnu.org/licenses/old-licenses/gpl-2.0.html](GNU Public License).
+This is considered "distribution" under the GPL. When distributing binaries, 
+or object code, covered under the GPL "complete and corresponding" source code 
+is required to be provided as well. We comply with these obligations 
+by publishing source code on GitHub and through our continuous integration 
+system (CI).
+
+In order to comply with all FOSS software licenses, public CI systems must 
+plublish complete and corresponding source code in conjunction with the publishing
+of object code. This include a yocto sstatecache, binary images for hardware 
+targets, and binary packages, like rpms or debs.
+
+Decision
+--------
```

Comment:
I don't think we should have headings that are about decisions. The net outcome of whatever decisions that have been made during development should be documented instead. In some cases a rationale is relevant to provide context and guidance for projects though.